edifact-edi/2.1/modules/ROOT/pages/index.adoc

```diff
@@ -135,8 +135,8 @@ The values are:
 |Key |Description
 |idRef |The referenced segment ID. This is optional and verified if provided, but otherwise ignored. The position value is used to uniquely identify segments within the section.
 |position |The segment position within the message section.
-|usage |Usage code, which may be M for Mandatory, O for Optional, C for Conditional, or U for Unused.
-|count |Maximum repetition count value, which may be a number or the special value `>1` meaning any number of repeats. The count value is optional and a base definition value is used if a value is not specified.
+|usage |Usage code, which can be M for Mandatory, O for Optional, C for Conditional, or U for Unused.
+|count |Maximum repetition count value, which can be a number or the special value `>1` meaning any number of repeats. The count value is optional and a base definition value is used if a value is not specified.
```

Comment:
Add a comma before "meaning" (global, at least for what you changed).

I suggest making the last sentence two sentences.